components/layout.js

```diff
@@ -21,6 +22,10 @@ export default function Layout({ children }) {
 
       <Header />
 
+      {bannerText && bannerTitle ? (
+        <Banner siteTitle={bannerTitle} headline={bannerText} />
+      ) : null}
```

Comment:
This is kind of weird, I assumed we would pass in the component itself, not just pass attributes to layout. Could we not just pass it in on the new page?

Reply:
This was how Alpha implemented it, tried doing it the way you mentioned prior to doing it this way and it was doing some funky stuff with the positioning/size of the banner (either being too small or off to the right side of the screen).